docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -0,0 +1,397 @@
+| ``<defaultMulticastLocatorList>`` | List of default input multicast locators. |br|   | ``Locator``         |         |
+|                                   | It expects a :ref:`LocatorListType`.             |                     |         |
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+| ``<sendSocketBufferSize>``        | Size in bytes of the output socket buffer. |br|  | ``uint32``          | 0       |
+|                                   | If the value is zero then Fast DDS will use |br| |                     |         |
+|                                   | the default size sockets configuration, |br|     |                     |         |
+|                                   | using a minimum size of 65536 bytes.             |                     |         |
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+| ``<listenSocketBufferSize>``      | Size in bytes of the input socket buffer. |br|   | ``uint32``          | 0       |
+|                                   | If the value is zero then Fast DDS will use |br| |                     |         |
+|                                   | the default size sockets configuration, |br|     |                     |         |
+|                                   | using a minimum size of 65536 bytes.             |                     |         |
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+| ``<builtin>``                     | |DomainParticipantQosWireProtocolBuiltin|        | :ref:`builtin`      |         |
+|                                   | public data member of the |br|                   |                     |         |
+|                                   | |DomainParticipantQosWireProtocolClass|          |                     |         |
+|                                   | class. |br|                                      |                     |         |
+|                                   | See the                                          |                     |         |
+|                                   | :ref:`builtin` section.                          |                     |         |
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+| ``<port>``                        | Allows defining the port and gains related |br|  | `Port`_             |         |
+|                                   | to the RTPS protocol. See the `Port`_ section.   |                     |         |
++-----------------------------------+--------------------------------------------------+---------------------+---------+
+| ``<participantID>``               | DomainParticipant's identifier. Typically |br|   | ``int32``           | 0       |
```

Comment:
Which data member is this?

Reply:
eprosima::fastrtps::rtps::RTPSParticipantAttributes::participantID

Reply:
Ese cambió de sitio. Es el principal API break

Reply:
Forget it. It was domain id...